app.py

```diff
@@ -1,40 +1,74 @@
 from flask import Flask, render_template, request, redirect, url_for
 import session_items as session
+import logging
+from classes import todoStatus, todoItem
+from trello_api import trelloGet, get_trello_lists, get_trello_cards, trelloPost, trelloPut, trelloDelete
 from operator import itemgetter
 
 app = Flask(__name__)
+# uncomment to debug API calls
+# logging.basicConfig(level=logging.DEBUG)
 app.config.from_object('flask_config.Config')
 
 
 @app.route('/', methods=['GET'])
-def get_todo_list():
-    todo_list = session.get_items()
-    newlist = sorted(todo_list, key=itemgetter('status'), reverse=True)
-    return render_template('index.html', items=newlist)
+def get_trello_todo_list():
+    trello_todo_list = get_trello_cards()
+    app.logger.info('Processing get cards request')
+    return render_template('index.html', items=trello_todo_list)
 
 
 @app.route('/create', methods=['POST'])
-def create():
-    session.add_item(request.form['add_todo'])
+def new_todo():
+    trelloPost(request.form['add_todo'], request.form['add_desc'])
+    app.logger.info('Processing create new card request')
     return redirect('/')
 
 
 @app.route('/update', methods=['POST'])
 def update():
-    for id in request.form:
-        new_status = request.form.get(id)
-        if new_status == 'Delete':
-            session.delete_item(id)
+    for trelloId in request.form:
```

Comment:
Obviously this will slow down if there are many cards on the board as you're making a separate HTTP request for every single card whether or not its status needs to change. 

You don't need to change this now, but just making sure you're aware.